js/controller/BoardController.js

```diff
@@ -45,6 +45,13 @@ app.controller('BoardController', function ($rootScope, $scope, $stateParams, St
 
 	$scope.board = BoardService.getCurrent();
 	StackService.clear(); //FIXME: Is this still needed?
+	$scope.setPageTitle = function() {
+		if(BoardService.getCurrent()) {
+			document.title = BoardService.getCurrent().title + " - Deck - " + oc_defaults.name;
```

Comment:
It would look a bit better with a different divider between board title and app title, such as `·` or `|`